### ISSUE TITLE
No Getter for skippedAsNotFailingStatus

### ISSUE THREAD

**Issue by BALyons** [jenkinsci/cucumber-reports-plugin#415]
https://github.com/jenkinsci/cucumber-reports-plugin/blob/22a9358af8675bcfd2d12cb943322dd7e2babc4c/src/main/java/net/masterthought/jenkins/CucumberReportPublisher.java#L342 

The field skippedAsNotFailingStatus does not have a getter.  I'm guessing the line above is meant to bet a 'get' and not a 'set'. 

Here's the error that I'm seeing regarding this

```
WARNING org.jenkinsci.plugins.structs.describable.DescribableParameter uncoerce failed to uncoerce net.masterthought.jenkins.CucumberReportPublisher@2034c95
java.lang.UnsupportedOperationException: no public field ‘skippedAsNotFailingStatus’ (or getter method) found in class net.masterthought.jenkins.CucumberReportPublisher
	at org.jenkinsci.plugins.structs.describable.DescribableParameter.getValue(DescribableParameter.java:161)
	at org.jenkinsci.plugins.structs.describable.DescribableParameter.inspect(DescribableParameter.java:142)
	at org.jenkinsci.plugins.structs.describable.DescribableModel.uninstantiate2(DescribableModel.java:655)
	at org.jenkinsci.plugins.structs.describable.DescribableModel.uninstantiate2_(DescribableModel.java:752)
	at org.jenkinsci.plugins.structs.describable.DescribableParameter.uncoerce(DescribableParameter.java:196)
	at org.jenkinsci.plugins.structs.describable.DescribableParameter.inspect(DescribableParameter.java:142)
	at org.jenkinsci.plugins.structs.describable.DescribableModel.uninstantiate2(DescribableModel.java:655)
	at org.jenkinsci.plugins.workflow.actions.ArgumentsAction.resolve(ArgumentsAction.java:306)
	at org.jenkinsci.plugins.workflow.actions.ArgumentsAction.getResolvedArguments(ArgumentsAction.java:293)
	at org.jenkinsci.plugins.workflow.cps.nodes.StepAtomNode.getDelegateType(StepAtomNode.java:134)
	at org.jenkinsci.plugins.workflow.cps.nodes.StepAtomNode.effectiveDisplayName(StepAtomNode.java:90)
	at org.jenkinsci.plugins.workflow.cps.nodes.StepAtomNode.getTypeDisplayName(StepAtomNode.java:102)
	at org.jenkinsci.plugins.workflow.graph.FlowNode.getDisplayName(FlowNode.java:258)
	at com.cloudbees.workflow.rest.external.FlowNodeExt.addBasicNodeData(FlowNodeExt.java:206)
	at com.cloudbees.workflow.rest.external.AtomFlowNodeExt.create(AtomFlowNodeExt.java:71)
	at com.cloudbees.workflow.rest.external.ChunkVisitor.makeAtomNode(ChunkVisitor.java:68)
	at com.cloudbees.workflow.rest.external.ChunkVisitor.atomNode(ChunkVisitor.java:154)
	at org.jenkinsci.plugins.workflow.graphanalysis.ForkScanner.fireVisitChunkCallbacks(ForkScanner.java:749)
	at org.jenkinsci.plugins.workflow.graphanalysis.ForkScanner.visitSimpleChunks(ForkScanner.java:767)
	at org.jenkinsci.plugins.workflow.graphanalysis.ForkScanner.visitSimpleChunks(ForkScanner.java:631)
	at org.jenkinsci.plugins.prometheus.util.FlowNodes.getSortedStageNodes(FlowNodes.java:26)
	at org.jenkinsci.plugins.prometheus.JobCollector.processPipelineRunStages(JobCollector.java:377)
	at org.jenkinsci.plugins.prometheus.JobCollector.processRun(JobCollector.java:356)
	at org.jenkinsci.plugins.prometheus.JobCollector.appendJobMetrics(JobCollector.java:264)
	at org.jenkinsci.plugins.prometheus.JobCollector.lambda$collect$0(JobCollector.java:204)
	at org.jenkinsci.plugins.prometheus.util.Jobs.forEachJob(Jobs.java:14)
	at org.jenkinsci.plugins.prometheus.JobCollector.collect(JobCollector.java:197)
	at io.prometheus.client.CollectorRegistry$MetricFamilySamplesEnumeration.findNextElement(CollectorRegistry.java:190)
	at io.prometheus.client.CollectorRegistry$MetricFamilySamplesEnumeration.nextElement(CollectorRegistry.java:223)
	at io.prometheus.client.CollectorRegistry$MetricFamilySamplesEnumeration.nextElement(CollectorRegistry.java:144)
	at io.prometheus.client.exporter.common.TextFormat.write004(TextFormat.java:22)
	at org.jenkinsci.plugins.prometheus.service.DefaultPrometheusMetrics.collectMetrics(DefaultPrometheusMetrics.java:54)
	at org.jenkinsci.plugins.prometheus.service.PrometheusAsyncWorker.execute(PrometheusAsyncWorker.java:41)
	at hudson.model.AsyncPeriodicWork.lambda$doRun$1(AsyncPeriodicWork.java:102)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

Thank you!

**Comment by damianszczepanik** [comment]
Very funny bug :)

**Comment by sujithmathew93** [comment]
Hello @damianszczepanik, what we have seen with this newly added checkbox is that the value is not retained in the Jenkins Configuration. So once the checkbox is checked and the job configuration is saved, opening it again we are seeing that the checkbox has not retained this selection, hence not able to use the 'Mark skipped as not failing status' option.
Has this been addressed in the latest version of the plugin 5.7.3?

**Comment by damianszczepanik** [comment]
Testing last version with that fix and could not reproduce the problem
